### DISCUSSION TITLE
Keep reviewer team listed on PR even after approval

### DISCUSSION THREAD

**Discussion by Baggerone** [community/community#12252] [Pull Requests]
Currently on a PR, when a team is the only reviewer listed and someone approves the PR, that user's name replaces the team in the list of reviewers.

I think it would be helpful for the team to remain at the bottom of the list of reviewers even if someone has already approved the PR.  That way, if more than one approval/review is desired, other people on the team will know that they are potential reviewers for it.

**Comment by erickaufman-surescripts** [comment]
+1, this should at least be configurable if not the default behavior.

We use team slugs with CODE_OWNERS to require at least one reviewer via a ruleset. 1 reviewer is the minimum we'd require, we often like to see more reviewers. When the team is removed they no longer see the PR under https://github.com/pulls/review-requested making it less likely more than the 1 approver can find or review the PR.

**Comment by guigomcha** [comment]
No progress or proposed workaroung this yet?

**Comment by erickaufman-surescripts** [comment]
A work around for this to enable auto assignment for the team: https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment

This will add individual members in addition to or instead of the team.

Caveat: the drop down for "How many team members should be assigned to review?" only goes up to 7, so unsure if it will work for teams with members counts greater than that. I was able to insert my own option to in the dropdown (10) and that persisted, but I did not test with a team count greater than 7.

**Comment by MuradMamedov** [comment]
The workaround isn't actually a true workaround as it doesn't provide the functionality which is wanted (especially when we have bigger or many teams).

Edit: I miss so many PRs because of this
